ui/app/src/Views/SilenceForm/Types.elm

```diff
@@ -172,12 +172,13 @@ defaultDuration =
     2 * Time.hour
 
 
-fromMatchersAndTime : List Matcher -> Time -> SilenceForm
-fromMatchersAndTime matchers now =
+fromMatchersAndTime : Maybe String -> List Matcher -> Time -> SilenceForm
+fromMatchersAndTime defaultCreator matchers now =
     { empty
         | startsAt = initialField (timeToString now)
         , endsAt = initialField (timeToString (now + defaultDuration))
         , duration = initialField (durationFormat defaultDuration)
+        , createdBy = initialField (Maybe.withDefault "" defaultCreator)
```

Comment:
Sorry, I just thought that we don't really need `defaultCreator` to be a `Maybe String` and we can always keep it as an empty string if it's not set. This would make the code slightly simpler, and the decoder would be:

```
    |> Json.decodeValue (Json.field "defaultCreator" Json.string)
    |> Result.withDefault ""
```